src/components/Center/centerHeader.js

```diff
@@ -1,19 +1,17 @@
 import ProfilePicture from '../../assets/me.jpg'
+import back from '../../assets/back.svg'
+import forward from '../../assets/forward.svg'
 
 export function CenterHeader(){
   return(
     <div className='main-viev-header'>
           <header className='main-header'>
             <div className='navi-button'>
               <button className='main-header-btn-left'>
-                <svg height='16' width='16' viewBox='0 0 16 16'>
-                  <path d='M 11.03 0.47 a 0.75 0.75 0 0 1 0 1.06 L 4.56 8 l 6.47 6.47 a 0.75 0.75 0 1 1 -1.06 1.06 L 2.44 8 L 9.97 0.47 a 0.75 0.75 0 0 1 1.06 0 Z'></path>
-                </svg>
+                <img src={back} />
```

Comment:
Try to do it that way:
https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs